wordpress-mobile/WordPress-iOS
# Update WordPressKit & WordPressAuthenticator to use the latest versions

Fixes # n/a

### To test:
1. `rake dependencies`
2. run & build the app
3. Navigate to a site 
4. Set Network Link Conditioner to 100% Loss
5. Navigate to Activity
6. If testing in Xcode, set a breakpoint in ActivityServiceRemote, L49
7. If testing on a device, check the app log for this message:

```
Check the console.log or app log for NSLocalizedDescription=The request timed out., NSErrorFailingURLStringKey=https://public-api.wordpress.com/wpcom/v2/sites/70760568/activity?_locale=en&number=1000&page=1, NSErrorFailingURLKey=https://public-api.wordpress.com/wpcom/v2/sites/{site_id}/activity?_locale=en&number=1000&page=1, _kCFStreamErrorDomainKey=4
```

### Do not merge until:
1. WordPressKit PR has been merged https://github.com/wordpress-mobile/WordPressKit-iOS/pull/232 ✔️
2. WordPressKit has the newly published release, `4.6.0-beta.6`  ✔️
3. Authenticator PR has been merged https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/201  ✔️
4. Authenticator has the newly published release, `1.11.0-beta.6`  ✔️
5. The podfile has been updated to WordPressKit 4.6.0-beta.6 and Authenticator 1.11.0-beta.6  ✔️
6. The NSURL+IDN pod changes have been merged https://github.com/wordpress-mobile/WordPress-iOS/pull/13644

PR submission checklist:
